workflows/raredisease.nf

```diff
@@ -215,12 +217,22 @@ workflow RAREDISEASE {
         ch_versions = ch_versions.mix(ch_sv_annotate.versions)
     }
 
-    // STEP 2.1: MT CALLING
+    // STEP 2.1: PREPARING MT ALIGNMENT
```

Comment:
I think we should consider moving the entire MT part of the pipeline to it's own subworkflow - which then calls the other subworkflows (aling, call, annotate etc ).

Reply:
Done :)